src/Plugin/StatisticsPlugin/Dummy.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Drupal\sapi\Plugin\StatisticsPlugin;
+
+use Drupal\sapi\Plugin\StatisticsPluginInterface;
+
+/**
+ * @StatisticsPlugin(
+ *  id = "dummy",
+ *  label = "Dummy label"
+ * )
+ */
+class Dummmy implements StatisticsPluginInterface {
```

Comment:
`StatisticsPluginInterface` implements `PluginInspectionInterface` and to comply with that the plugin needs to have `getPluginId()` and other methods which are provided by `PluginBase` class which is a parent of `StatisticsPluginBase`. So, unless the Plugin really wants to implement all those declared classes, it should extend `StatisticsPluginBase` or at least `PluginBase`.